apps/masters/models.py

```diff
@@ -1,8 +1,63 @@
+import os
+import uuid
 from django.db import models
+from django.dispatch import receiver
+from django.db.models.signals import pre_delete
 from utils_methods import *
 from utils_variables import *
 
 # Create your models here.
+class Country(models.Model):
+    country_id = models.AutoField(primary_key=True)
+    country_name = models.CharField(max_length=100)
+    country_code = models.CharField(max_length=10)
+
+    def __str__(self):
+        return self.country_name
+    
+    class Meta:
+        db_table = countrytable
+
+class State(models.Model):
+    state_id = models.AutoField(primary_key=True)
+    country_id = models.ForeignKey(Country, on_delete=models.CASCADE, null=True, default=None, db_column = 'country_id')
+    state_name = models.CharField(max_length=100)
+    state_code = models.CharField(max_length=10)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
This should satisfy below statement, did you test?
updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP

Reply:
yes tested Thulasi  it is satisfied.